Source/gmenu.cpp

```diff
@@ -415,11 +415,12 @@ void gmenu_slider_1(TMenuItem *pItem, int min, int max, int gamma)
 
 int gmenu_slider_get(TMenuItem *pItem, int min, int max)
 {
-	int v3;          // eax
-	unsigned int v4; // ecx
+	int v3, v4;

```

Comment:
v3 is the number of steps that the scroll bar is divided in to.
v4 is the current step that the scroll bar is set to.
```suggestion
	int nSteps, step;
```